src/battle/ov16_02268520.c

```diff
@@ -45,9 +45,9 @@ static const SpriteTemplate Unk_ov16_022700CC[] = {
 };
 
 /*
-* Maps a battle platform ID to the NARC member in pl_batt_obj containing a sprite for it.
-*/
-__attribute__((aligned(4))) static const u16 ov16_BattlePlatformIndexToNARCMember[] = {
+ * Maps a battle terrain index to the NARC member in pl_batt_obj containing a sprite for it.
+ */
+__attribute__((aligned(4))) static const u16 ov16_BattleTerrainIndexToNARCMember[] = {
```

Comment:
**polish**: Same suggestion about types/classes applies here; there is a set of terrain constants defined in `consts/terrain.json`, which gets compiled to `build/consts/terrain.h`.

...and looking at the names there, it will likely be better for us to align the names there with the names from these terrain blobs.